src/CoreBundle/Event/ResourceCollectionJavaScriptCollectedEvent.php

```diff
@@ -13,13 +13,20 @@
 
 use Symfony\Contracts\EventDispatcher\Event;
 
+/**
+ * @psalm-suppress InvalidReturnStatement, InvalidReturnType
+ * @FIXME Default value of `$content` is an empty array, when everything outside of this class expects `getContent` to return a string.
+ */
 class ResourceCollectionJavaScriptCollectedEvent extends Event implements ResourceCollectionJavaScriptCollectedEventInterface
 {
     /**
-     * @var string
+     * @var array|string
      */
     private $content = array();
```

Comment:
The default value of `$content` does not make a lot of sense: It is expected to be a string everywhere in this class, but is an empty array by default if `null` was passed to the constructor.